config.yml

```diff
@@ -42,6 +42,12 @@ defaults:
       secretAccessKey:  !env AWS_SECRET_ACCESS_KEY
       region:           us-west-2
       apiVersion:       2014-01-01
+  pulse:
+    username:       !env PULSE_USERNAME
+    password:       !env PULSE_PASSWORD
+    hostname:       !env PULSE_HOSTNAME
+    vhost:          !env PULSE_VHOST
+
```

Comment:
It isn't directly relevant to this PR, but you will probably want to use tc-lib-pulse, rather than PulseListener, when you start listening for arbitrary pulse messages.

Reply:
yeah, I can use these [here](https://github.com/taskcluster/taskcluster-lib-pulse#client) as raw AMQP creds, right?

Reply:
Yep.  But let's not do that in this PR -- this one is close enough to land-able already :)